src/main/java/com/afornalik/ox/Match.java

```diff
@@ -1,63 +1,64 @@
-package com.afornalik.ox.controller;
+package com.afornalik.ox;
 
-import com.afornalik.ox.model.board.Board;
-import com.afornalik.ox.model.board.FieldStatus;
-import com.afornalik.ox.model.board.OutOfBoardException;
-import com.afornalik.ox.model.player.Player;
-import com.afornalik.ox.model.player.PlayerContainer;
 import com.afornalik.ox.view.UIExtended;
 
-class MatchController {
+class Match {
 
     private final UIExtended uiExtended;
     private final Board board;
     private final PlayerContainer playerContainer;
+    private BoardChecker boardChecker;
 
-    MatchController(UIExtended uiExtended, Board board, PlayerContainer playerContainer) {
+    Match(UIExtended uiExtended, Board board, PlayerContainer playerContainer) {
         this.uiExtended = uiExtended;
         this.board = board;
         this.playerContainer = playerContainer;
+        boardChecker = new CheckHorizontally(board);
```

Comment:
O ile we wczesnej wersji można mieć sprawdzacza tylko poziomego, o tyle miej na uwadze, że w końcowej potrzebujesz też pionowego, przekątnego i antyprzekątnego.
I mając to na uwadze... na ile jeden mecz jest testowalny, jeśli w środku ma 4 sprawdzaczy (w konstruktorze - 4 wołania konstruktorów)?

Reply:
Spróbuję utworzyć klasę pomocniczą, w której zamknę cały proces sprawdzania. Ta klasa będzie miała instancję wszystkich sprawdzaczy oraz metodę zwracającą bula. Jeszcze nad tym pracuję.